CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * [Sync] `SyncConfiguration` and `SyncSession` has been moved to `io.realm.mongodb.sync`.
 * [Sync] `EmailPasswordAuth` has been movedto `io.realm.mongodb.auth`.
 * [Sync] Improved exception hiearchy for App and Sync exceptions. All sync/app exceptions now use `io.realm.mongodb.exceptions.AppException` as their top-level exception type. Many methods have more specialized exceptions for common errors that can be caught and reacted to. See `AppException` documentation for more details.
+* [Sync] `SyncConfiguration.directory` is no longer available and `AppConfiguration.syncRootDirectory` has been added instead to allow users to set the root folder containing all files used for data synchronization between the device and MongoDB Realm. (Issue [#795](https://github.com/realm/realm-kotlin/issues/795))
```

Comment:
``SyncConfiguration.directory` is no longer available` this should move to `Breaking Changes`

Reply:
I'm not sure what you mean by moving this to breaking changes. This is the breaking changes section.

I will move the part about `AppConfiguration.syncRootDirectory` to the `Fixed` section.

Reply:
Ups 🙈 ... for some reason I thought this was under Enhancements.